src/test/unittest/unittest.sh

```diff
@@ -1996,6 +2020,10 @@ function check() {
 # pass -- print message that the test has passed
 #
 function pass() {
```

Comment:
This function prints the 'pass' message.
Also, there might be test scripts that do not call this function (and instead exit from the script).

I think we should have an explicit acquire/release API for devices.

And for the future, the test framework should be redesigned to accept a list of supported persistence "providers" and the tests would be called for every possible permutation of those "providers". This would eliminate the need to manually specify devdax tests and we could automatically schedule test execution so that those providers are never used concurrently, for example, by having a N test queues, where each queue is responsible for one provider type. If no providers are present, other queues are scheduled for execution - we wouldn't have to wait on a lock.